docs/more/faq.rst

```diff
@@ -140,4 +129,34 @@ What operating systems do you support?
 We currently support Linux and Mac OS X.
 We have initial Windows support, and expect publish a release for Windows soon.
 
-Sign up for our Newsletter if you'd like to stay up to speed on the latest product updates.
+Sign up for our newsletter in the footer anywhere on our docs if you'd like to
+stay up to speed on the latest product updates.
+
+--------------------------
+What license does oso use?
+--------------------------
+
+.. |apache2| replace:: the *Apache 2.0 license*
+.. _apache2: https://github.com/osohq/oso/blob/master/LICENSE
+
+oso licensed under |apache2|_.
+
+----------------------
+How does pricing work?
+----------------------
+
+oso is freely available as an open source product and will always be open source. 
```

Comment:
This makes it sound like it will always be open source but not necessarily always free. We should clarify that oso will always be open source **and** free.